tests/test_cli/test_search.py

```diff
@@ -111,43 +131,37 @@ def setup_class(cls):
         cls.cwd = os.getcwd()
         cls.runner = CliRunner()
 
+    def _generated_expected_output(self):
```

Comment:
You're redefining the same function here.

Reply:
These are in different functions